unit3-server-mvc-with-node-express-and-sql/README.md

```diff
@@ -2,50 +2,58 @@
 
 ### Week 1
 
+- Serverside MVC with Express: 3 days
+  - Why MVC for server side development: Separation of concerns
+  - Routing & (simple) regular expressions
+  - Controllers
+  - Views using a templating engine and ViewModels (Mustache? Jade?)
```

Comment:
Yeah good call...will be good to introduce them to simple server side templating. Let's go with templating engine and then we can always touch on some dead simple react view stuff if there's time/interest